DigitalOffice/Services/CompanyService/CompanyService.csproj

```diff
@@ -7,6 +7,10 @@
     </PropertyGroup>
 
     <ItemGroup>
+      <PackageReference Include="MassTransit" Version="7.0.2" />
+      <PackageReference Include="MassTransit.AspNetCore" Version="7.0.2" />
+      <PackageReference Include="MassTransit.Extensions.DependencyInjection" Version="7.0.2" />
+      <PackageReference Include="MassTransit.RabbitMQ" Version="7.0.2" />
```

Comment:
Where  installed greenpipes library?

Reply:
@djino97 GreenPipes are not necessary. 
My setup works perfectly without GreenPipes.

Reply:
Yes, it works without greenpipes